firebase.json

```diff
@@ -346,6 +346,8 @@
       { "source": "/go/floating-snackbar-offset", "destination": "https://docs.google.com/document/d/1elP-y83PtvfAZHNcpHCtnOFhZO9VnnlobwfQ33QO4hg/edit", "type": 301 },
       { "source": "/go/floating-widgetspans", "destination": "https://docs.google.com/document/d/1I-VqCxvszXGAas_6EE9b6ZIzyMY3HjxDxGW9_DWWd7M/edit", "type": 301 },
       { "source": "/go/flutter-developer-tooling-plugins", "destination": "https://docs.google.com/document/d/189A-T-pcHPuds8pOtbzLS8Wl6Hr0AIAIl0aRTPvLKec/", "type": 301 },
+      { "source": "/go/flutter-devtools-extensions", "destination": "https://docs.google.com/document/d/189A-T-pcHPuds8pOtbzLS8Wl6Hr0AIAIl0aRTPvLKec/", "type": 301 },
+      { "source": "/go/flutter-devtools-static-extensions", "destination": "https://docs.google.com/document/d/1MQUPUe2Gs93g_Mbh6aDCMflQcr0Kv0PUEWZXHGF19fo/", "type": 301 },
```

Comment:
I can’t seem to access this one. Is it shared to anyone with the link?

Reply:
Hmm that's odd. Yes it is. I verified I can open it in an incognito window.

Reply:
Even if I sign in I can't access this one (https://docs.google.com/document/d/1MQUPUe2Gs93g_Mbh6aDCMflQcr0Kv0PUEWZXHGF19fo/). Does the link you're using in incognito have a resource key query parameter?

Reply:
I can't either. I am shown this error message.

![image](https://github.com/flutter/website/assets/706219/ef38ca50-847a-4e1c-b1ae-7769ba8fb7c9)